### ISSUE TITLE
Razer firefly not smooth color changing

### ISSUE THREAD

**Issue by woloss** [antonpup/Aurora#597]
### What are you experiencing an issue with?
- [ ] Aurora
- [ ] Aurora supported game, (Fill in the game)
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [X] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?
- [ ] Crash
- [ ] Performance problems
- [X] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [ ] Device issue
- [ ] Other, (Fill in another type of issue here)

### Expected behavior:
All device change colors w/o delays and totally smooth.

### Actual behavior:
My mouse and keyboard changing colors very smoothly. While Razer Firefly looks like turn off one led, wait 0.1-0.15 sec and turn on another one.
If set effect speed to x10. Razer firefly will delay with color so much. (keyboard + mouse at green, while razer firefly at red).
No problem when aurora turned off. And no problem if aurora is on and logitech software is off.
I did something yesterday and everything was smooth, but today again. I tried a lot of things, nothing work.

Logitech devices: G910 + G102
Razer: Razer Firefly

### How often does this problem occur?
This problem occurs everytime

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. Enable Gradient fill layer
2. Set to Rainbow
3. Effect Speed to x10

### What is the version of Aurora you are experiencing this issue with?
My version is... v0.6.0b

### Does the previous version of Aurora have this issue?
The previous version, didn't try previous ones.

**Comment by woloss** [comment]
Ok. i tried more. If assign half keys of keyboard, no problems. Maybe there is a way to send more resources for aurora?

**Comment by simon-wh** [comment]
So, when you stopped using USB 3.1 the issue went away? Or only after you assigned half the keys on the keyboard?

**Comment by woloss** [comment]
After stop using USB 3.1 the issue is gone.

Only one problem I still have, if setup Solid Color Layer. Sometimes Razer Firefly blink for 1 sec. If use Breathing layer this happens too, but a bit less.

**Comment by simon-wh** [comment]
I think this seems to a problem with the Razer SDK and/or the device, not Aurora.
